Formula/kubecfg.rb

```diff
@@ -0,0 +1,48 @@
+class Kubecfg < Formula
+  desc "Manage complex enterprise Kubernetes environments as code"
+  homepage "https://github.com/ksonnet/kubecfg"
+  url "https://github.com/ksonnet/kubecfg/archive/v0.7.2.tar.gz"
+  sha256 "a01de014904af9177bb0ad15a249346d4b7bd6412268d0f83756541aa19fa38c"
+  head "https://github.com/ksonnet/kubecfg.git"
+
+  depends_on "go" => :build
+  depends_on "go-bindata" => :build if build.head?
+
+  def install
+    # Standard gopath shenanigans to keep "go build" happy:
+    sources = buildpath.children - [buildpath/".brew_home"]
+    ENV["GOPATH"] = buildpath
+    srcdir = buildpath/"src/github.com/ksonnet/kubecfg"
+    srcdir.mkpath
+    mv sources, srcdir
+    # The real build steps:
+    cd srcdir do
+      args = []
+      args.push("VERSION=#{version}") unless build.head?
+      system "make", "generate", *args if build.head?
```

Comment:
`make generate` (and go-bindata) are only required at dev-time, and the up-to-date generated file is _supposed_ to be already included in git repo.  .. So I think you should remove this step (and go-bindata dependency) from the homebrew formula.

Reply:
Simpler is better, so I'll do that.